### PR TITLE
Add Animals Table

### PR DESCRIPTION
**Pull request by Munsa1**
# Add Animals Table

### In the following milestone, I:

- Create a table named animals with the following columns:

1. id: integer

2. name: string

3. date_of_birth: date

4. escape_attempts: integer

5. neutered: boolean

5. weight_kg: decimal

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,40 @@
-# vet-clinic-database
\ No newline at end of file
+# vet-clinic-database
+
```
Comment (SajjadAhmad14) [inline README.md:2]: Please add the project description in the README file. A professional README should have a project title, description, and a set of instructions to use the project locally.

File: queries.sql [hunk 2]
```diff
@@ -0,0 +1,17 @@
+/* Add Queries in-line with the requirements */
+
+SELECT * from animals where animal_name LIKE '%mon';
+
+SELECT animal_name FROM animals WHERE date_of_birth >= '2016/01/01' AND date_of_birth <= '2019/12/31';
+
+SELECT animal_name FROM animals WHERE neutered = 'true' AND escape_attempts <= '3';
```
Comment (SajjadAhmad14) [inline queries.sql]: Please list the name of all animals that have less than 3 escape attempts not equal to and less than 3 attempts.

**Review by SajjadAhmad14 — CHANGES_REQUESTED** [review]
## Required Changes ♻️
> "Change is inevitable. Change is constant. -Benjamin Disraeli"
## Hi Munsa, 
 ### You have been doing a tremendous job so far :ok_hand:

## To Highlight 🎉
PR has a title and summary. :heavy_check_mark:
Git flow is followed. :heavy_check_mark:
## To Improve 🙏

- [ ] Please check inline comments under the review

## Optional suggestions
_Comments with the [OPTIONAL] prefix are not a must to implement however, It's strongly recommended to work on them as they can make your code better._

## Happy coding!👏👏👏

#### Please leave any questions or comments in the PR if you have any doubts or unclarity!

**_Note: Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: queries.sql (hunk 2 above)
Comment (Munsa1) [reply queries.sql]: Thanks for the feedback, issues have been resolved

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED 🎉🍾✔**
Hello @Munsa1 👋
Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**